apps/thunderhub/docker-compose.yml

```
@@ -0,0 +1,27 @@
+version: "3.7"
+
+x-logging:
+  &default-logging
+  driver: journald
+  options:
+    tag: "umbrel-app {{.Name}}"
+
+services:
+  thub:
+    image: apotdevin/thunderhub:v0.12.0
+    logging: *default-logging
+    restart: on-failure
+    stop_grace_period: 5m
+    ports:
+      - 3000:3000
+    volumes:
+      - ${LND_DATA_DIR}:/lnd:ro
+    environment:
+      NO_VERSION_CHECK: "true"
+      NO_CLIENT_ACCOUNTS: "true"
+      LOG_LEVEL: "debug"
+      SSO_SERVER_URL: "https://${LND_IP}:${LND_GRPC_PORT}"
+      SSO_MACAROON_PATH: "${LND_DATA_DIR}/data/chain/bitcoin/${BITCOIN_NETWORK}"
+      SSO_CERT_PATH: "${LND_DATA_DIR}/tls.cert"
+      DANGEROUS_NO_SSO_AUTH: "true"
+      LOGOUT_URL: "http://umbrel-dev.local"
```

You should add compatibility with the current Umbrel URL.
```suggestion
      LOGOUT_URL: ${DEVICE_HOSTS:-"http://umbrel.local"}
```

---

`$DEVICE_HOSTS` is an array of all the possible URLs and IP addresses that the user could be accessing their Umbrel at. It wouldn't exactly tell you which URL the user is currently at (umbrel.local, Tor hidden service, or IP address) plus we don't expose this variable to apps (as it can leak the dashboard's hidden service address).

It would perhaps be better to drop the redirect logic entirely, 'cause even though you can get the IP address/hostname via `window.hostname` in the browser on local network to correctly figure out the right redirect URL, you won't be able to figure out dashboard's onion address if the user is accessing via Tor.